src/Evolution/Systems/Cce/AnalyticBoundaryDataManager.hpp

```diff
@@ -68,6 +68,11 @@ class AnalyticBoundaryDataManager {
 
   size_t get_l_max() const noexcept { return l_max_; }
 
+  const std::unique_ptr<Solutions::WorldtubeData>& get_generator()
```

Comment:
Do you actually need the `unique_ptr`, or can you just return a reference to the contained value?

Also, I think this should be in the "Allow non-monotonic steps in RT analytic solution for self-start" commit.